app/legacy/entitlements.js

```diff
@@ -0,0 +1,11 @@
+const { get } = require('../api')
+
+const getEntitlements = async (organisationId, callerId) => {
+  const url = `/SitiAgriApi/entitlements/grouped/${organisationId}`
+  const data = await get(url, callerId)
+  return data?.data.reduce((acc, { quantityOwned }) => acc + quantityOwned, 0)
```

Comment:
Suggest returning `0` as default if `data` is undefined.
ie.
`return data?.data.reduce((acc, { quantityOwned }) => acc + quantityOwned, 0) ?? 0`